app/assets/stylesheets/admin/views/_document-collection-group-memberships-index.scss

```diff
@@ -15,7 +15,7 @@
   display: flex;
   align-items: center;
 
-  & > p:first-child {
+  & > li:first-child {
     @include govuk-responsive-padding(2, "right");
     @include govuk-responsive-margin(2, "right");
     border-right: 1px solid $govuk-border-colour;
```

Comment:
I think it would be better to lose the whole `app-view-document-collection-group-memberships-index__heading-links` class and add a more generic style nested in the `app-view-document-collection-group-memberships-index__heading` element like below. 

That way we can just allow the `<li>` elements to flow more naturally and add the separating border in a way that doesn't rely on a specific number of links to be present. It also centres the vertical rule better and keeps the horizontal alignment with the header. 

I think I'd also lose the flexbox layout in the containing element as well because that makes it display oddly on smaller viewports. By floating the `<ul>` element we can let it flow underneath the heading in this case, though we do need to change the heading from a block level element to allow this, and give it a bit of a margin to avoid the elements clashing. 

Once we're nested inside the very specific `app-view-document-collection-group-memberships-index__heading` element I think we can afford to be far more generic in the way we reference elements so I don't think we need to add specific classes for those. 

See what you think. I know it's quite a big change but it makes it closer to the sort of styles we are using in related components. (Also I think flexbox can be overused. 😄)

```
.app-view-document-collection-group-memberships-index__heading {
  .gem-c-heading {
    display: inline-block;
    @include govuk-responsive-margin(2, "right");
  }

  .govuk-list {
    float: right; 

    li {
      display: inline-block;
      @include govuk-responsive-padding(2, "left");
      @include govuk-responsive-margin(2, "left");
      border-left: 1px solid $govuk-border-colour;

      &:first-child {
        margin-left: 0;
        padding-left: 0;
        border-left: none;
      }
    }
  }
}
```